data/locale/de.ts

```diff
@@ -5735,7 +5735,7 @@ PM bedeutet Phasen-Modulation: Die Phase von Oszillator 3 wird durch Oszillator
     </message>
     <message>
         <source>Timeline controls</source>
-        <translation>Zeitlinien Regler</translation>
+        <translation>Zeitlinienregler</translation>
```

Comment:
The better translation for timeline would be "Zeitstrahl", "Zeitachse" or "Zeitleiste". Also, the [Duden](https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich) recommends a minus between composed words to increased readability.

Asides, this translation occurs twice in that file. Can you please fix both?